Tools/i18n/makelocalealias.py

```diff
@@ -135,8 +135,8 @@ def check(data):
     args = parser.parse_args()
 
     data = locale.locale_alias.copy()
-    data.update(parse_glibc_supported(args.glibc_supported))
     data.update(parse(args.locale_alias))
+    data.update(parse_glibc_supported(args.glibc_supported))
```

Comment:
IMO, this is the only interesting change. I only upgraded the alias list because finding an old version of glibc is slightly annoying and updating it is unlikely to break things.